azurelinuxagent/common/protocol/wire.py

```diff
@@ -900,70 +900,36 @@ def _call_hostplugin_with_container_check(self, host_func):
                              log_event=True)
                 raise
 
-    def __send_request_using_host_channel(self, host_func):
```

Comment:
Same thing here: __send_request_using_host_channel() and __send_request_using_direct_channel() attempted to provide a uniform interface to download report errors. They are no longer needed, since all the download functions use exceptions.